wp-content/plugins/core/src/Blocks/Patterns/Pattern_Category.php

```diff
@@ -4,15 +4,23 @@
 
 class Pattern_Category {
 
-	public const CUSTOM_PATTERN_CATEGORY_SLUG = 'tribe_custom';
+	public const CUSTOM_PATTERN_CATEGORIES = [
+		[ 'slug' => 'cta', 'name' => 'Calls to Action' ],
+		[ 'slug' => 'cards', 'name' => 'Cards' ],
+		[ 'slug' => 'headers', 'name' => 'Headers' ],
+		[ 'slug' => 'media', 'name' => 'Media' ],
+		[ 'slug' => 'templates', 'name' => 'Templates' ],
+	];
 
 	public function register_pattern_category(): void {
-		register_block_pattern_category(
-			self::CUSTOM_PATTERN_CATEGORY_SLUG,
-			[
-				'label' => esc_html__( 'Custom', 'tribe' ),
-			]
-		);
+		foreach ( self::CUSTOM_PATTERN_CATEGORIES as $category ) {
+			register_block_pattern_category(
+				$category['slug'],
+				[
+					'label' => esc_html__( $category['name'], 'tribe' ),
```

Comment:
You don't want to use variables for translation.  It does work programmatically but when trying to do the translations, the software will just show `$category['nam']` and not the value from the array.

Reply:
I had Justin help me rewrite - should look better now!